client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/screen-category-list.tsx

```diff
@@ -114,7 +114,11 @@ const ScreenCategoryList = ( {
 							} }
 						>
 							<span>{ label }</span>
-							<Icon icon={ chevronRight } size={ 24 } />
+							<Icon
+								className="screen-category-list__arrow-icon"
+								icon={ chevronRight }
```

Comment:
I saw Gutenberg uses `isRTL()` to determine which icon they want to use according to the direction

https://github.com/WordPress/gutenberg/blob/trunk/packages/components/src/navigation/item/index.tsx#L73

So, it might be better to use `isRTL()` for the icon from `@wordpress/icons` to align with Gutenberg. WDYT?

Reply:
We can also use `useRtl` from `i18n-calypso`. So many options 😅

Reply:
>I saw Gutenberg uses isRTL() to determine which icon they want to use according to the direction

I thought it would be better, both in terms of cohesiveness and performance, to guarantee the appearance-related things on the CSS side, but I don't have a strong opinion about this!

>We can also use useRtl from i18n-calypso. So many options 

I searched with `.rtl`, `isRTL`, and `useRtl` in Calypso, and it seems that, democratically, `.rtl` is the most used 😇